src/componets/Form/Form.tsx

```diff
@@ -0,0 +1,337 @@
+    alignItems: "center",
+    justifyContent: "center",
+  };
+
+  const ButtonStyle = {
+    width: "10%",
+    padding: "10px",
+    marginTop: "10px",
+    border: "none",
+    borderRadius: "5px",
+    marginRight: "10%",
+  };
+  const BoxButton = {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-evenly",
+    alignItems: "center",
+    width: "100%",
+  };
+
+  const RadioGroupStyle = {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-around",
+    alignItems: "center",
+  };
+
+  return (
+    <Box component="form" onSubmit={handleSubmit} sx={BoxFormStyle}>
+      <Typography
+        variant="h5"
+        sx={{ textAlign: "center", color: "primary.main" }}
+      >
+        Car adding form
+      </Typography>
+      <TextField
+        label="Brand"
+        name="brand"
+        value={formData.brand}
+        onChange={handleInputChange}
+        required
+        sx={TextFieldStyle}
+      />
+      <TextField
+        label="Model"
+        name="model"
+        value={formData.model}
+        onChange={handleInputChange}
+        required
+        sx={TextFieldStyle}
+      />
+      <TextField
+        label="Year"
+        name="year"
+        type="number"
+        value={formData.year}
+        onChange={handleInputChange}
+        required
+        InputProps={{ inputProps: { min: 2000, max: 2024 } }}
+        sx={TextFieldStyle}
+      />
+      <FormControl required sx={TextFieldStyle}>
+        <InputLabel>Body Type</InputLabel>
+        <Select
+          name="body_type"
+          value={formData.body_type}
+          onChange={handleSelectChange}
+          sx={{ width: "100%" }}
+        >
+          <MenuItem value="">
+            <em>Select...</em>
+          </MenuItem>
+          {bodyTypes.map((type) => (
+            <MenuItem key={type} value={type}>
+              {type}
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
+      <TextField
+        label="Mileage"
+        name="mileage_km"
+        type="number"
+        value={formData.mileage_km}
+        onChange={handleInputChange}
+        required
+        InputProps={{ inputProps: { min: 1, max: 1000 }, endAdornment: <InputAdornment position="end">1,000 km</InputAdornment> }}
+        sx={TextFieldStyle}
+      />
+      <FormControl component="fieldset" required sx={TextFieldStyle}>
+        <FormLabel component="legend">Gearbox</FormLabel>
+        <RadioGroup
+          name="gearbox"
+          value={formData.gearbox}
+          onChange={handleInputChange}
+          sx={RadioGroupStyle}
+        >
+          <FormControlLabel
+            value="Manual"
+            control={<Radio />}
+            label="Manual"
+            sx={RadioStyle}
+            required
+          />
+          <FormControlLabel
+            value="Automatic"
+            control={<Radio />}
+            label="Automatic"
+            sx={RadioStyle}
+          />
```

Comment:
You pass the same styles to this component many times. What do you think about creating a separate `RadioFormControlLabel` component, or something like that, that has these default styles and contains the control as Radio?